dali/operators/reader/loader/video/video_loader_decoder_base.h

```diff
@@ -36,6 +36,7 @@ class VideoSample {
  public:
   Tensor<Backend> data_;
   int label_;
+  int first_frame_;
```

Comment:
Since `VideoSample` already has a non-trivial constructor, some initialization (e.g. to invalid values, like -1) would be nice.

Reply:
Fixed